assets/css/main.scss

```diff
@@ -278,3 +278,7 @@ ul.li {
   font-style: italic;
   cursor: pointer;
 }
+
+a.external:after {
+  content: url("/img/Icon_External_Link.png");
```

Comment:
Maybe use ➚ instead since it's unicode and so should copy automatically when @bitschmidty makes the newsletter email (I believe he needs to copy any images?).  Its HTML entity coding is `&#x279a;`